ftw/publisher/sender/tests/tests.zcml

```diff
@@ -5,6 +5,7 @@
     <include package="Products.Five" file="meta.zcml" />
     <include package="Products.Five.viewlet" file="meta.zcml" />
     <include package="Products.GenericSetup" file="meta.zcml" />
+    <include package="plone.resource" file="meta.zcml" />
```

Comment:
I have the suspicion that this file (`tests.zcml`) is no longer used as we have removed all PloneTestCase tests. Could you verify that?

Reply:
The file is still used in https://github.com/4teamwork/ftw.publisher.sender/blob/42dae51919c9482d556f4d2f3e6df6b4a69a41de/ftw/publisher/sender/testing.py#L23 to create a `ZCML_LAYER` .

The `ZCML_LAYER` in turn is still used in https://github.com/4teamwork/ftw.publisher.sender/blob/42dae51919c9482d556f4d2f3e6df6b4a69a41de/ftw/publisher/sender/tests/test_meta_zcml.py#L14